quickstart/docker/all-in-one/README.md

```diff
@@ -42,7 +42,9 @@ This is the primary use case for this project: running the `ziti edge quickstart
     docker compose logs --follow
     ```
 
-5. Run the CLI inside the quickstart environment.
+5. Open the console in a browser: [localhost:1280/zac/](https://localhost:1280/zac/)
```

Comment:
should have more detail. if they change ${ZITI_CTRL_ADVERTISED_PORT} this is wrong